tests/lcs/core/engine/widget.livecodescript

```diff
@@ -80,3 +81,21 @@ end TestWidgetScriptObjectAccess
 on errorDialog pError
    put pError into gLastSentWidgetError
 end errorDialog
+
+//////////
+
+on TestWidgetThrowInOnCreate
+   set the lockErrorDialogs to true
+   create stack "WidgetThrowOnSaveTest"
+   set the defaultStack to "WidgetThrowOnSaveTest"
+
+   put empty into gLastSentWidgetError
+   
+   create widget "Test" as "com.livecode.lcs_tests.core.widgetthrow"
```

Comment:
At the moment any other execution error in this script will be captured and ignored. If it's all handled centrally then it's not an issue as you would tell the test lib the name of the error to expect for a test and it would treat all others the way it does now.